src/Catty/Extension&Delegate&Protocol/Extensions/UIViewController/ProjectDetailStoreViewControllerExtension.swift

```diff
@@ -72,4 +72,51 @@ extension ProjectDetailStoreViewController {
         }
         return InputValidationResult.validInput()
     }
+
+    func download(name: String) {
+        self.storeProjectDownloader.download(
+            projectId: self.project.projectID,
+            completion: { _, storeProjectDownloaderError in
+                if let error = storeProjectDownloaderError {
+                    switch error {
+                    case .unexpectedError, .timeout:
+                        Util.defaultAlertForNetworkError()
+                    case .parse(error: _), .request(error: _, statusCode: _):
+                        Util.alert(withText: kLocalizedInvalidZip)
+                    }
+                    self.setBackDownloadStatus()
+                    return
+                }
+
+                self.storeProjectDownloader.setProjectNameForDownload(projectName: self.project.projectName)
+
+                DispatchQueue.main.async(execute: {
+                    Util.showNotification(withMessage: kLocalizedDownloaded)
```

Comment:
Please remove this notification and the `kLocalizedDownloaded`.

Reply:
Removed!